lib/libc/string/wcscasecmp.c

```diff
@@ -35,11 +35,13 @@ wcscasecmp(const wchar_t *s1, const wchar_t *s2)
 {
 	wchar_t c1, c2;
 
-	for (; *s1; s1++, s2++) {
+	for (;; s1++, s2++) {
```

Comment:
There's no real reason for this change and the companion if below.

Reply:
Yes there is because if the leading bit is 1, the return value will be positive instead of negative.